peachjam/views/courts.py

```diff
@@ -118,10 +119,16 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context["court"] = self.court
         context["registry_label_plural"] = CourtRegistry.model_label_plural
-        context["registries"] = self.court.registries.exclude(
+        context["registries"] = registries = self.court.registries.exclude(
             judgments__isnull=True
         )  # display registries with judgments only
-        context["registry_groups"] = list(chunks(context["registries"], 2))
+        # split the list in the middle to have two columns and preserve ordering
```

Comment:
is a template update necessary?

Reply:
no, I've changed the context variable to work with what's there